### ISSUE TITLE
NTR: Synovial biopsy: mycobacteria by culture (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8084]
New term request Synovial biopsy: mycobacteria by culture (athrey_140304135240):

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000734
- label: Positive synovial mycobacterial culture
- parents: [Positive synonvial fluid culture (HP:6000582)](https://hpo.jax.org/app/browse/term/HP:6000582)
- pmid: 
- def: Growth of a mycobacteria in a culture derived from synovial fluid.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8084
- synonyms:
